examples/homography_estimation.py

```diff
@@ -4,11 +4,14 @@
 # LICENSE file in the root directory of this source tree.
 
 import glob
+import logging
 import os
 import shutil
-from typing import List
+import warnings
+from typing import Dict, List, Tuple, cast
 
 import cv2
+import hydra
```

Comment:
The examples/readme needs updating to reflect that this example also need hydra to be installed.